src/types.ts

```diff
@@ -277,11 +284,19 @@ export type ControlOptions = {
   country?: string | string[];
 
   /**
-   * Filter of feature types to return.
+   * Types to query. Array of types or `[minZoom, maxZoom, type]` items, where `minZoom` is inclusive, `maxZoom` exclusive and can be `null` or `undefined` for unbound.
    *
-   * Default value is `undefined` - all available feature types are returned.
+   * Default value is `undefined` - server default feature types
    */
-  types?: string[];
+  types?: TypeRule[];
+
+  /**
+   * Same as `types` but used for reverse geocoding.
+   *
+   * Default value is value of `types`.
+   */
+  // TODO use reasonable types (see GO-729)
```

Comment:
I don't think this is good to have for everyone to see in the autocomplete in their IDE :)

Reply:
`//` coment is not JSDoc and in my IDE (vscode) it does not appear. Which IDE do you use?

Reply:
oh sorry, I missed it's out of the jsdoc